File: Deja_Vu/Deja_Vu.java
import java.util.Scanner;

public class Program
{
public static boolean dub(String str)
{
    int strz = str.length();
    for(int x=0;x<strz;++x)
    {
        for(int i=x+1;i<strz;i++)
        {
  if(str.charAt(i) == str.charAt(x)) return true;
        }
    }
    return false;
}
 public static void main(String[] args) {
        Scanner inp = new Scanner(System.in);
        String word = inp.nextLine();
        if (dub(word))
        System.out.print("Deja Vu");
        else
        System.out.print("Unique");
    }
}

File: Deja_Vu/Deja_Vu.cpp
#include <iostream>
using namespace std;
bool dub(string str)
{
    int strz = str.length();
    for(int x=0;x<strz;++x)
    {
        for(int i=x+1;i<strz;i++)
        {
      if(str[i] == str[x]) return true;
        }
    }
    return false;
}

int main() {
string word ="";
cin >> word;
if(dub(word))
cout << "Deja Vu";
else
cout << "Unique";
    return 0;
}
